backstop/tests/user-icon.html

```diff
@@ -331,56 +317,56 @@ <h2 id="status">Status</h2>
             src="https://itwinplatformcdn.azureedge.net/iTwinUI/user-placeholder.png"
             alt="Terry Rivers"
           />
-        </span>
-      </div>
-
-      <div id="away">
-        <span
-          class="iui-user-icons-medium"
-          title="Robin Mercer"
-        >
+          <span class="iui-stroke"></span>
           <span
-            title="Away"
-            class="iui-user-icons-status iui-away"
+            title="Online"
+            class="iui-status iui-online"
           >
-            <!-- Use away_2.svg from generic-icons package. -->
+            <!-- Use checkmark.svg from generic-icons package. -->
             <svg
-              class="iui-user-icons-status-symbol"
+              class="iui-status-symbol"
               aria-hidden="true"
               viewBox="0 0 16 16"
             >
-              <path d="m13.445 12.832-6.445-4.297v-7.535h2v6.465l5.555 3.703z" />
+              <path d="M6,14L0,8l2-2l4,4l8-8l2,2L6,14z" />
             </svg>
           </span>
-          <span class="iui-user-icons-stroke"></span>
-          <abbr
-            class="iui-user-icons-initials"
-            style="background-color: #6AB9EC;"
-          >
-            RM
-          </abbr>
         </span>
+      </div>
 
+      <div id="away">
         <span
-          class="iui-user-icons-medium"
+          class="iui-user-icon"
           title="Robin Mercer"
         >
+          <abbr
+            class="iui-initials"
+            style="background-color: #6AB9EC;"
+          >
+            RM
+          </abbr>
+          <span class="iui-stroke"></span>
           <span
             title="Away"
-            class="iui-user-icons-status iui-away"
+            class="iui-status iui-away"
           >
             <!-- Use away_2.svg from generic-icons package. -->
```

Comment:
Maybe update everywhere the icons? Now half you updated, half not..

Reply:
Can we finalize #10? It's been sitting there since the beginning 😄

Reply:
All the icon names & code have been updated to match 1.0.2 of iTwinUI-icons.  Will try to get #10 either finalized or abandon it and start a new one depending on how much has changed since that initial PR.

Reply:
I guess it is not really generic-icons package anymore.. it is just icons or itwinui-icons or full name @itwin/itwinui-icons... Names of icons look ok, I guess.

Reply:
I was gonna say the same thing (it's a problem in all new PRs). But I think name of package can be changed when #10 gets finalized (or recreated)